docs/LearnCalico.md

```diff
@@ -0,0 +1,364 @@
+# Calico networking in a containerized environment
+
+This GitHub repository is the entry point for running a Calico network in a
+containerized environment.
+
```

Comment:
You're missing a general description of what Calico is (or at the very least a link somewhere), and what the actual setup looks like.  Could start by talking about multiple hosts, workloads etc.  We should have that level of detail in the main calico docs, so perhaps link to it.  Without that summary it feels a little odd jumping straight into hosts and workloads etc.